here-naksha-lib-model/src/commonMain/kotlin/naksha/model/request/PropertyFilter.kt

```diff
@@ -0,0 +1,89 @@
+package naksha.model.request
+
+import naksha.base.AnyObject
+import naksha.base.Platform
+import naksha.jbon.JbFeatureDecoder
+import naksha.model.request.query.*
+
+class PropertyFilter(val req: ReadFeatures) : ResultFilter {
+
+    companion object {
+        const val PROPERTIES = "properties"
+    }
+
+    override fun call(a1: ResultTuple): ResultTuple? {
```

Comment:
Ok, but what makes sense in the abstract entity as `Fn1`, might be more specifically named in more specific case ;)
Legality of overridden method does not require the name of the argument to be the same as super-method

Reply:
Is there a need for this? I would prefer that this `PropertyFilter` does not implement the interface, then we can call this arg `tuple`. But since it does, I assume there will be a chance this is used interchangably with other filters through the interface, and changing the name of the arg might be problematic, if the only advantage is code readability.
<img width="911" alt="image" src="https://github.com/user-attachments/assets/816c6f54-e25f-4c42-812a-1ef464158e25">

Reply:
I get both arguments, should we not simple change the `ResultFilter` to:
```kotlin
/**
 * A functional interface to a function lambda.
 */
@Suppress("OPT_IN_USAGE")
@JsExport
fun interface ResultFilter : Fn {
    fun matches(resultTuple: ResultTuple): Boolean
}
```
Or alike?

Reply:
Sounds good, I will do 
```
interface ResultFilter : Fn {
    fun filter(resultTuple: ResultTuple): ResultTuple?
}
```